jbpm-workitems/jbpm-workitems-rest/src/test/java/org/jbpm/process/workitem/rest/RestWorkItemHandlerTest.java

```diff
@@ -133,6 +136,34 @@ public void testGETOperation() {
         assertEquals("request to endpoint " + workItem.getParameter("Url") + " successfully completed OK",
                      responseMsg);
     }
+    
+    @Test
+    public void testGETWithProxyOperation() { 
+        System.setProperty(RESTWorkItemHandler.USE_SYSTEM_PROPERTIES, "true");
+        System.setProperty("http.proxyHost", "127.0.0.1");
```

Comment:
If I set this ip to an arbitrary one that does not exist. This test does not fail.
Looks to me that this is a faulty test.

Reply:
Test modified and moved to a new class